### PR TITLE
SDI-324 Add CMD tile to DPS & navigation link to CMD homepage

### PR DESCRIPTION
**Pull request by steverendell**
Add DPS link (and tidy unused env vars)

This provides a way back to DPS if the user arrived in CMD from the DPS tile

### CODE REVIEW

File: server/views/pages/calendar.njk [hunk 1]
```diff
@@ -53,6 +54,13 @@
       </div>
     {% endif %}
 
+    {{ govukBreadcrumbs({
+      collapseOnMobile: false,
```
Comment (petergphillips) [inline server/views/pages/calendar.njk:58]: shouldn't this be `true`?  On mobile users won't be able to access DPS.

Comment (steverendell) [reply server/views/pages/calendar.njk:58]: according to gov-uk docs, true just means it only shows the first and last items on mobile devices, so not applicable here

**Review by petergphillips — DISMISSED** [review]
LGTM

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**